README.md

```diff
@@ -65,16 +65,26 @@ docker pull xaionaro2/edk2-builder:RefindPlusUDK
 # hacky fix for duplication error of lodepng_malloc and lodepng_free
```

Comment:
I don't use Docker myself so can't verify, but have things actually been tried without this section ?
1. RefindPlus should not include any instances of lodepng_free or lodepng_malloc
2. lodepng_refit_free and lodepng_refit_malloc should be unique to RefindPlus

Unclear therefore why there should be any duplication errors to fix.

Reply:
Without it you get error:
```
"llvm-ar" cr /home/edk2/Build/RefindPlus/RELEASE_CLANG38/X64/RefindPlusPkg/RefindPlus/OUTPUT/RefindPlus.lib  @/home/edk2/Build/RefindPlus/RELEASE_CLANG38/X64/RefindPlusPkg/RefindPlus/OUTPUT/object_files.lst
Waiting for thread ending...(1)
Waiting for thread ending...(1)
"clang" -o /home/edk2/Build/RefindPlus/RELEASE_CLANG38/X64/RefindPlusPkg/RefindPlus/DEBUG/RefindPlus.dll -nostdlib -Wl,-n,-q,--gc-sections -z common-page-size=0x40 -Wl,--entry,_ModuleEntryPoint -u _ModuleEntryPoint -Wl,-Map,/home/edk2/Build/RefindPlus/RELEASE_CLANG38/X64/RefindPlusPkg/RefindPlus/DEBUG/RefindPlus.map,--whole-archive -flto -Wl,-Oz -Wl,-melf_x86_64 -Wl,--oformat=elf64-x86-64 -Wl,-pie -mcmodel=small -Wl,--start-group,@/home/edk2/Build/RefindPlus/RELEASE_CLANG38/X64/RefindPlusPkg/RefindPlus/OUTPUT/static_library_files.lst,--end-group -g -fshort-wchar -fno-builtin -fno-strict-aliasing -Wall -Werror -Wno-array-bounds -ffunction-sections -fdata-sections -include AutoGen.h -fno-common -DSTRING_ARRAY_NAME=RefindPlusStrings -Wno-parentheses-equality -Wno-tautological-compare -Wno-tautological-constant-out-of-range-compare -Wno-empty-body -Wno-unused-const-variable -Wno-varargs -Wno-unknown-warning-option -fno-stack-protector -mms-bitfields -Wno-address -Wno-shift-negative-value -Wno-unknown-pragmas -Wno-incompatible-library-redeclaration -fno-asynchronous-unwind-tables -mno-sse -mno-mmx -msoft-float -mno-implicit-float -ftrap-function=undefined_behavior_has_been_optimized_away_by_clang -funsigned-char -fno-ms-extensions -Wno-null-dereference -m64 "-DEFIAPI=__attribute__((ms_abi))" -mno-red-zone -mcmodel=small -fpie -Oz -flto -target x86_64-pc-linux-gnu -Wno-error -Os -DEFIX64 -D__MAKEWITH_TIANO -DREFIT_DEBUG=0 -DMDEPKG_NDEBUG -Wno-error -Wl,--defsym=PECOFF_HEADER_SIZE=0x228 -Wl,--script=/home/edk2/edk2/BaseTools/Scripts/GccBase.lds -Wno-error -O3
Waiting for thread ending...(1)
Waiting for thread ending...(1)
/usr/bin/ld: lodepng_xtra.obj (symbol from plugin): in function `lodepng_refit_malloc':
(.text+0x0): multiple definition of `lodepng_refit_malloc'; lodepng.obj (symbol from plugin):(.text+0x0): first defined here
/usr/bin/ld: lodepng_xtra.obj (symbol from plugin): in function `lodepng_refit_malloc':
(.text+0x0): multiple definition of `lodepng_refit_free'; lodepng.obj (symbol from plugin):(.text+0x0): first defined here
```

Reply:
@dakanji : I can pack the resulting environment where this error is reproducible and send to you. If you could take a look it would really help. What do you think?

Reply:
Will look into the issues with lodepng_refit_free and lodepng_refit_malloc and let @startergo know when ready.
You can eliminate the hacks for lodepng_free, lodepng_malloc and STATIC_ASSERT in the meantime.